client/src/index.js

```diff
@@ -155,32 +156,82 @@ export default function Login() {
     }
 
     const handleSubmit = () => {
+        //Login
         if (state.isLogin) {
-            handleLogin()
-            return
+            if (document.getElementById(document.getElementById("standard-password").value === "")) {
+                //setTransition();
+                //setOpen(true);
+                console.log("Please fill in all requirements!!!")
+            } else {
+                fetch(util.format('%s/api/login', config.EXPRESS_BACKEND), {
+                    method: "POST",
+                    headers: {
+                        'Content-type': 'application/json'
+                    },
+                    body: JSON.stringify(state)
+                })
+                .then(result => {
+                    console.log(result) // 401 = Unauthorized; 200 = OK
+                    if (result.ok) {
+                        // Handle successful login here
+                        //go to the Dashboard
+                        return 
+                    }
+                    // Handle non-successful login here
+                })
+            }
+        } else {
+            //Sign up
+            if (document.getElementById("standard-password").value === "" ||
+                document.getElementById("email").value === "" ||
+                document.getElementById("firstname").value === "" ||
+                document.getElementById("lastname").value === "" ||
+                document.getElementById("confirmPassword").value === "") {
+                //setTransition();
+                //setOpen(true);
+                console.log("Please fill in all requirements!!!")
+            } else {
+                fetch(util.format('%s/api/signup', config.EXPRESS_BACKEND), {
+                    method: "POST",
+                    headers: {
+                        'Content-type': 'application/json'
+                    },
+                    body: JSON.stringify(state)
+                })
+                .then(result => {
+                    console.log(result) // 500 = Internal Service Error; 201 = CREATED
+                    if (result.ok) {
+                        // Handle successful signup here
+                        return
+                    }
+                    // Handle non-successful signup here
+                })
+            }
```

Comment:
Why are you removing the `handleLogin()` function and adding code I already added?